make_helpers/armv7-a-cpus.mk

```diff
@@ -32,12 +32,17 @@ endif
 #
 # ARCH_SUPPORTS_LARGE_PAGE_ADDRESSING = 0/1
 # Defines if core supports the Large Page Addressing extension.
+#
+# ARCH_SUPPORTS_VIRTUALIZATION = 0/1
+# Defines if ARMv7 core supports the Virtualization extension.
 
 define pre-a15-extensions
 ARCH_SUPPORTS_LARGE_PAGE_ADDRESSING	?= 	0
+ARCH_SUPPORTS_VIRTUALIZATION		?= 	0
 endef
 define post-a15-extensions
 ARCH_SUPPORTS_LARGE_PAGE_ADDRESSING	?= 	1
+ARCH_SUPPORTS_VIRTUALIZATION		?= 	1
```

Comment:
The use of pre-processor build macros is currently a bit inconsistent in ARM TF (see https://github.com/ARM-software/tf-issues/issues/446). As part of cleaning this up, we would like to move to a model wherein a build macro, if used as a boolean flag, is either defined or not defined. 

Hence it would be good to have all new code follow this semantics and  `$(eval $(call add_define,ARCH_SUPPORTS_VIRTUALIZATION))` should be done here. The code would then change to explicitly check if the macro is defined or not using `defined()` pre-processor directive.

Could you migrate all the new macros as suggested above ?

Also, suggest to rename the macro to indicate these are ARMv7 specific like `ARMV7_VIRTUALIZATION_SUPPORT`